### ISSUE TITLE
Focus in not getting back to menu context once clicked on escape button from full screen.

### ISSUE THREAD

**Issue by SreejaPampara** [highcharts/highcharts#16339]
#### Expected behaviour
With Maintaining Focus the Menu button should also opened when clicked enter Once clicked on escape button from full screen.

#### Actual behaviour
Highlighting the Menu button but focus is somewhere else, and the menu button is not getting opened by clicking enter button once coming back by clicking escape button from full screen.
Observed both for Graphs and pie Chart.

#### Live demo with steps to reproduce
https://jsfiddle.net/SreejaPampara/hj7x3w2q/1/

#### Product version
HighCharts version : v9.2.2
#### Affected browser(s)
Chrome, Edge

**Comment by raf18seb** [comment]
Hi @SreejaPampara,

Exiting from the full screen by the `escape` button click is a **default browser action** - it's not coming from the Highcharts library.

You can add the possibility to open fullscreen on the enter button click e.g. this way (very simple example): https://jsfiddle.net/BlackLabel/j5uar1vx/

I'm tagging this ticket as a `feature request`, please vote for it by adding a thumb up 👍

**Comment by SreejaPampara** [comment]
Hi @raf18seb is it possible to maintain access the menu icon after returning from full screen as the menu icon is highlighted as focus is present there as it is causing a accessibility issue. Attaching the screenshots for reference.
    
![image](https://user-images.githubusercontent.com/88161368/133588960-73d1d9a9-0fcc-44b2-803a-c2c146c4cbec.png)

- As in full screen also it is taking additional tab to go to menu.

![image](https://user-images.githubusercontent.com/88161368/133588318-75e44757-f382-4f97-95d8-0c63b5308077.png)

**Comment by raf18seb** [comment]
Hi @SreejaPampara,

I think I misunderstood you in the first post.
This could be improved - the exporting button is focused after exiting from the fullscreen, but the exporting menu is not being opened when clicking enter (it's working fine before entering the fullscreen and it's working fine after exiting from the fullscreen but pressing the additionat `tab` button is required.

**Comment by SreejaPampara** [comment]
Hi @KacperMadej, 
 Yes. Thank you for the response. Could you please let me know  for this workaround, any code changes need to be done from our end, or else the existing version of this will be included with this workaround.

**Comment by raf18seb** [comment]
@SreejaPampara, is the workaround provided by @KacperMadej working fine for you? if not, could you tell us what could be improved?
https://jsfiddle.net/BlackLabel/j5uar1vx/2/

**Comment by vaishnavi-115** [comment]
Hi @KacperMadej 
The temporary fix suggested was not sufficient for the testing team. They have also checked with their SME on this. Please suggest a code fix or another work around for this issue.
Also it would be helpful if we can get an estimated date, by which this issue will be fixed.

**Comment by oysteinmoseng** [comment]
@vaishnavi-115 I will take a look at this issue as soon as possible and provide a stable fix. Expect this to be resolved by early next week.

**Comment by vaishnavi-115** [comment]
Hi @oysteinmoseng
Thank you for providing the fix.
Please let us know if any code changes are required from our end for this fix or please provide the latest version in which these changes are included.

**Comment by vaishnavi-115** [comment]
As we know that HIGHCHART is NOT open source software.
Could you please give us the details about which Microsoft team owns it currently.
Thank you.

**Comment by oysteinmoseng** [comment]
Hi @vaishnavi-115, we can not share license information publicly on GitHub. Please [get in touch](https://shop.highsoft.com/contact) with our sales team and they will be happy to assist you with any questions related to licensing.
